common/util/sys_util.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+import (
+	"time"
+)
+
+// GetCurrentTimeMillis returns the time in millis for now.
+func GetCurrentTimeMillis() int64 {
+	return time.Now().UnixNano() / time.Millisecond.Nanoseconds()
```

Comment:
Is it better to make this function return float? float is more precise and then this function can be reused in metrics.
```
return float64(time.Now().UnixNano()) / float64(time.Millisecond.Nanoseconds())
```

Reply:
Maybe it's better to wrap it as another function, otherwise we need to make a lot of changes.

Reply:
Make sense.